src/components/App/App.jsx

```diff
@@ -1,6 +1,32 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from 'react-router-dom';
+import ApiPage from '../Api/ApiPage.jsx';
+import DetailPage from '../Detail/DetailPage.jsx';
+import Header from '../Header/Header.jsx';
+import ApiPageAll from '../Api/ApiPageAll.jsx';
 
 export default function App() {
-  return <h1>Hello World</h1>;
+  return (
+    <>
+      <Router>
+        <Header />
+        <Switch>
+          <Route exact path="/">
```

Comment:
You ApiPage component is not working and it's breaking your app. It doesn't seem like you need it. So, maybe you could use your ApiPageAll component at the "/" path instead.